### ISSUE TITLE
Alpaca showing balances too high

### ISSUE THREAD

**Issue by BenLeibig** [farm-army/farm-army-backend#21]
If you go to the farming page of an open farming posistion you see something like this:

![image](https://user-images.githubusercontent.com/11885129/114071870-e68ba480-98a1-11eb-8164-39e1e99dea1e.png)

Note the 3 BNB amounts.  Posistion, debt and equity.  The posistion is the total being farmed, the debt is how much is being borrowed to achieve that, and the equity is the difference, which is how much your posistion is actually worth if you withdraw.

Right now with FARM.army shows this:
![image](https://user-images.githubusercontent.com/11885129/114072025-12a72580-98a2-11eb-9020-579f3cd00fdf.png)
which is 93 BNB which is the same as the posistion value shown on the site.

Basically you need to either grab the "equity" value of bnb, or subtract the debt from the posistion in order to get the actual equity.

Drop me a comment if you want a wallet address, but I think this is clear and should be easy to fix.

**Comment by Haehnchen** [comment]
"equity" is now calculated, feedback is welcome

**Comment by BenLeibig** [comment]
Looks perfect.  Thanks a lot.  Awesome project!
